avm/utilities/pipelines/sharedScripts/Set-EnvironmentOnAgent.ps1

```diff
@@ -258,21 +258,4 @@ function Set-EnvironmentOnAgent {
     }
 
     Write-Verbose ('Install-CustomModule end') -Verbose
-
```

Comment:
Instead of removing this code I'd suggest parameterizing it and disabling it by default. As discussed it may be that this comes handy again in the future.

Reply:
done